src/Summary/SummaryManager.php

```diff
@@ -52,6 +54,26 @@ public function removeExperience(Adherent $adherent, JobExperience $experience):
         return true;
     }
 
+    public function updateTrainings(Summary $summary, Training $training): void
+    {
+        $summary->addTraining($training);
+        $this->updateSummary($summary);
+    }
+
+    public function removeTraining(Adherent $adherent, Training $training): bool
+    {
+        if (!$summary = $this->repository->findOneForAdherent($adherent)) {
+            return false;
+        }
+
+        SummaryItemDisplayOrderer::removeItem($summary->getTrainings(), $training);
```

Comment:
This could probably be encapsulated instead of using static (perhaps for another PR after the CV).